website/docs/r/synthetics_monitor_step.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_synthetics_step_monitor"
+sidebar_current: "docs-newrelic-resource-synthetics-step-monitor"
+description: |-
+Create and manage a Synthetics monitor in New Relic.
+---
+
+# Resource: newrelic\_synthetics\_step\_monitor
+
+Use this resource to create, update, and delete the synthetics step monitor in New Relic.
+
+-> **NOTE:** The [newrelic_synthetics_private_location](newrelic_synthetics_private_location.html) resource private minion can take upto 10 minutes to be available through Terraform.
+
+## Example Usage
+
+##### Type: `STEP MONITOR`
+```hcl
+resource "newrelic_synthetics_step_monitor" "foo" {
+  name = "foo"
+  enable_screenshot_on_failure_and_script = true
+  locations_public = ["US_EAST_1", "US_EAST_2"]
+  period = "EVERY_6_HOURS"
+  status = "ENABLED"
+  steps {
+    ordinal = " "
```

Comment:
Can we add more information to the steps? Ideally a few steps with different examples.